src/features/controls.ts

```diff
@@ -217,6 +227,54 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
         {type: String, attribute: 'field-of-view', hasChanged: () => true})
     fieldOfView: string = DEFAULT_FIELD_OF_VIEW;
 
+    @style({
+      intrinsics: {
+        basis: [
+          numberNode(-Infinity, 'rad'),
+          numberNode(Math.PI / 8, 'rad'),
+          numberNode(0, 'm')
+        ],
+        keywords: {auto: [null, null, null]}
```

Comment:
It would probably be useful to add an `infinity` keyword here for convenience. Maybe:

```javascript
keywords: {
  auto: [null, null, null],
  infinity: [numberNode(-Infinity, 'rad'), numberNode(0, 'rad'), numberNode(0, 'm')]
}
```

WDYT?

Reply:
I dunno, specifying Infinity and getting zero seems pretty weird to me. The other reason I didn't go this route is because it goes by so many names: Inf, inf, infinity, -Inf, etc. Since auto does what we want anyway, it seemed simpler.

Reply:
Fair enough, we can always add it later